labview/labview_rabbitmq.c

```diff
@@ -10,144 +10,268 @@
 #include <stdio.h>
 
 
-MgErr copyStringToLStrHandle(char* cpString, LStrHandle LVString)
+MgErr copyStringToLStrHandle(char *cpString, LStrHandle LVString)
```

Comment:
pure reformatting, so i am not reviewing

Reply:
Probably its linter (code format) change, as I prefer to use "type*".